code/game/machinery/doors/airlock.dm

```diff
@@ -1173,7 +1173,10 @@
 
 			if(!density)//already open
 				return
-
+			if(!HAS_TRAIT(user, TRAIT_KNOW_ENGI_WIRES))
+				user.balloon_alert(user, "don't know wires!")
+				to_chat(user, span_warning("You don't know which wires to avoid when forcing open the airlock safely!"))
```

Comment:
You don't need both a balloon alert and a to_chat message, just pick one, probably the former.